src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -73,7 +73,7 @@ public class RemoteUriWebService : IRemoteUriWebService {
                         SetResponseHeaders(response, context.Response, localBaseUrl, remoteBaseUrl);
                         using (Stream respStream = response.GetResponseStream())
                         using (Stream outStream = context.Response.OutputStream) {
-                            await respStream.CopyAndFlushAsync(outStream);
+                            await respStream.CopyAndFlushAsync(outStream, ct);
```

Comment:
Async call inside finally block? @int19h , @karthiknadig , is it really required?